fastify.js

```diff
@@ -333,8 +333,15 @@ function fastify (options) {
         else lightMyRequest(httpHandler, opts, cb)
       })
     } else {
-      return this.ready()
-        .then(() => lightMyRequest(httpHandler, opts))
+      return lightMyRequest((req, res) => {
+        this.ready(function (err) {
+          if (err) {
+            res.emit('error', err)
```

Comment:
To test this part it should be enough to register a broken plugin:

```
fastify.register((instance, opts, next)=>{}) // not call next

try{
await fastify.inject().get()
}catch(err){
t.ok(err) // plugin timeout
}
```

Reply:
Added test with plugin that explicitly returns error; otherwise we'd have to wait until plugin initialization times out, which takes some time, resulting in slower tests. Do you see any disadvantages to that approach?

Reply:
No, you had a better idea 👍